common/fe-ste-models.go

```diff
@@ -119,6 +119,19 @@ func (DeleteSnapshotsOption) None() DeleteSnapshotsOption    { return DeleteSnap
 func (DeleteSnapshotsOption) Include() DeleteSnapshotsOption { return DeleteSnapshotsOption(1) }
 func (DeleteSnapshotsOption) Only() DeleteSnapshotsOption    { return DeleteSnapshotsOption(2) }
 
+type SetPropertiesAPIOption uint32 // [0000000000...32 times]
+
+var ESetPropertiesAPIOption = SetPropertiesAPIOption(0)
+
+// functions to set values
+func (SetPropertiesAPIOption) None()        { ESetPropertiesAPIOption |= 0 }
+func (SetPropertiesAPIOption) SetTier()     { ESetPropertiesAPIOption |= 1 }
+func (SetPropertiesAPIOption) SetMetadata() { ESetPropertiesAPIOption |= 2 }
+
+// functions to get values (to be used in sde)
+func (SetPropertiesAPIOption) TransferTier() bool     { return int32(ESetPropertiesAPIOption&1) == 1 }
```

Comment:
Having functions like this to check that it does one thing or another is a good idea for bitflag enums. However, I would advise 

1) Using a receiver variable (e.g. `func (e SetPropertiesAPIOption)`) and checking against that
2) Using the references to the actual flag names rather than magic numbers